azure-pipelines.yml

```diff
@@ -668,7 +668,12 @@ jobs:
       brew uninstall azure-cli
       
       echo == Install azure-cli.rb formula ==
-      brew install --build-from-source $SYSTEM_ARTIFACTSDIRECTORY/homebrew/azure-cli.rb
+      # Need to create a dummy homebrew tap to install the formula
```

Comment:
This method is found in https://github.com/Homebrew/brew/issues/18371#issuecomment-2365396463

Setting `HOMEBREW_DEVELOPER=1` is a another workaround but the owner says it changes a lot of the behaviour of Homebrew: https://github.com/Homebrew/brew/pull/20414#issuecomment-3183734076